charts/fybrik/values.yaml

```diff
@@ -50,6 +50,26 @@ global:
   # zerolog verbosity level 
   # ref: https://github.com/rs/zerolog#leveled-logging
   loggingVerbosity: -1
+  # True if the target deployment is OpenShift. Otherwise false. Default value is false.
+  openshift: false
+  # Pod Security Context. This is the default setting for all pods, and can be
+  # overwritten by a specific podSecurityContext settings.
+  # ref: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context
+  podSecurityContext:
```

Comment:
Is there a specific use case you are trying to support by allowing the pod security context and container security context to be configurable? Typically these are not configurable as they define the workload’s security posture.

Reply:
This is an open source project and we leave fine security tuning to our users.